README.md

```diff
@@ -1,3 +1,115 @@
+
+HMS Ewon Ignition Integration Demo was developed by the HMS Solution Center using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+
+## Issues/Problems ##
+If you experience a problem or bug, please check the Ewon Ignition Integration Demo Wiki for information, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/wiki](https://github.com/hms-networks/FlexyIgnitionDemo/wiki "hms-networks/FlexyIgnitionDemo/wiki")  
+
+If no information is available on the Ewon Ignition Integration Demo Wiki, or if your issue persists, please create a GitHub issue, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "hms-networks/FlexyIgnitionDemo/issues")  
+
+## Getting Started ##
+To complete installation, please perform all the setup steps in the sections below! To begin, the following items are required:  
+
+- HMS Ewon Product  
+- Computer with Administrator Privileges
+- Demo Resource Pack (.zip)
+	- Download Here: [https://github.com/hms-networks/FlexyIgnitionDemo/releases](https://github.com/hms-networks/FlexyIgnitionDemo/releases "https://github.com/hms-networks/FlexyIgnitionDemo/releases")
+
+## Ewon Setup ##
+*During Ewon setup, you will ensure that a user account with the correct permissions is available for use with the Ewon Ignition Integration Demo.*
+
+**Step 1:** If your Ewon device is not setup, use eBuddy and a web browser to complete the `System`, `Internet` and `VPN` wizards. Detailed instructions can be found in section 5 or 6 of your product's installation guide.
+
+- *Flexy 205*: [https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22](https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22 "https://websupport.ewon.biz/sites/default/files/ig-0028-00-en-installation-guide-for-flexy-205.pdf#page=22")
+- *Flexy Base Unit*: [https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28](https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28 "https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28")
+- *Cosy 131*: [https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24](https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24 "https://websupport.ewon.biz/sites/default/files/ig-0022-00-en-installation-guide-for-cosy-131.pdf#page=24")
```

Comment:
Cozy is not compatible with datamailbox.  Only the Flexy will work with ignition.

Reply:
Removed Cozy Link